UnrealPlugin/ThirdPersonMPGSDKSetup.md

```diff
@@ -0,0 +1,315 @@
+# ThirdPersonMP Example - GSDK Project Setup
```

Comment:
Keep some basic instructions in the readme.

All detailed information needs to be moved into the docs. This helps you to maintain the docs over time. We're not doing any docs tracking from readme files.

Reply:
You're probably looking for a level of nuance that I don't fully grasp. I will need some help to apply this guidance.

Reply:
Let's schedule a time in Dec to move this set of docs in the PlayFab docs. We don't have a TOC structure here in the readme docs. It'll be confusing to read everything here since there are multiple pages.